tools/image_processing/yolo/yolo_prediction.xml

```diff
@@ -0,0 +1,100 @@
+<tool id="yolo_predict" name="YOLO Prediction" version="@VERSION@+galaxy0" profile="24.2">
+    <description>Prediction with YOLO models</description>
+    <macros>
+        <import>macros.xml</import>
+    </macros>
+    <expand macro="creator" />
+    <requirements>
+	<container type="docker" >quay.io/sunyi000/ultralytics-galaxy:latest</container>
```

Comment:
```suggestion
	<container type="docker">quay.io/sunyi000/ultralytics-galaxy:latest</container>
```

Reply:
Where is this Dockerfile? Should we put it into a simliar setup as the IT? With CI and so on?